package.json

```diff
@@ -20,7 +20,7 @@
     "test"
   ],
   "engines": {
-    "node": ">=10.0"
+    "node": ">=10.17"
```

Comment:
`unhandled-rejections=strict` that allows us to throw when unhandled promise rejection happens is only available since 10.17